### PR TITLE
Refine cond API Chinese Doc for 2.0RC

### PR DESCRIPTION
**Pull request by zhhsplendid**
As the title, screenshot:

image

### CODE REVIEW

File: doc/paddle/api/paddle/fluid/layers/cond_cn.rst [hunk 1]
```diff
@@ -22,23 +22,25 @@ PaddlePaddle里Tensor的嵌套结构是指一个Tensor，或者Tensor的元组
        .. code-block:: python
                   
             import paddle.fluid as fluid
```
Comment (liym27) [inline doc/paddle/api/paddle/fluid/layers/cond_cn.rst]: import paddle.fluid as fluid -> import paddle

File: doc/paddle/api/paddle/fluid/layers/cond_cn.rst [hunk 2]
```diff
@@ -22,23 +22,25 @@ PaddlePaddle里Tensor的嵌套结构是指一个Tensor，或者Tensor的元组
...
-            a = fluid.data(name='a', shape=[-1, 1], dtype='float32')
-            b = fluid.data(name='b', shape=[-1, 1], dtype='float32')
+
+            paddle.enable_static()
+            a = paddle.data(name='a', shape=[-1, 1], dtype='float32')
+            b = paddle.data(name='b', shape=[-1, 1], dtype='float32')
```
Comment (Aurelius84) [inline doc/paddle/api/paddle/fluid/layers/cond_cn.rst]: paddle.static.data?

Comment (zhhsplendid) [reply doc/paddle/api/paddle/fluid/layers/cond_cn.rst]: Done

File: doc/paddle/api/paddle/fluid/layers/cond_cn.rst (hunk 1 above)
Comment (zhhsplendid) [reply doc/paddle/api/paddle/fluid/layers/cond_cn.rst]: Done

**Review by Aurelius84 — APPROVED** [review]

**Review by liym27 — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**